index.html

```diff
@@ -1,13 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./style/style.css">
-  <title>Killy's Portfolio</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Internal CSS File -->
+    <link rel="stylesheet" href="./style/style.css" />
+    <!-- External CSS File -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <title>Killy's Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav id="logo" class="dark-blue">
+        <a href="index.html">Welcome</a>
+      </nav>
+      <nav id="menu">
+        <i class="fa-solid fa-bars dark-blue"></i>
+      </nav>
+    </header>
+    <section id="headline">
+      <div id="headline-intro">
+        <h4>
+          Hey there. I'm Killy. <br />
+          <span class="sea-green">I'm a software developer</span>
+        </h4>
```

Comment:
- [ ] Please replace the `<h4>` tag with a `<h1>` tag because this is your main header on the page.

Reply:
@Omar-Muhamad  Thanks for the review. The reason I chose to use h4 instead of h1 was to achieve the look of the original portfolio project from Figma. using h1 would give a result of this
![h1-output](https://user-images.githubusercontent.com/89058561/164366287-613187be-18f6-48ed-82c4-f9fb0eaf8973.png)
.

whereas originally I used <p> tags for that and I got the original look as this
![ptags-outcome](https://user-images.githubusercontent.com/89058561/164366377-f56de912-a2c2-459b-b5bc-99424c1326cf.png)

but then web hint flagged an issue that sections need headings and h4 was the only heading that worked to achieve that look so I went with it

Reply:
hello 👋
I think you can control each aspect of your font throw styling like font size, font weight and line height.
And you can also divide this heading into two parts one with h1 tag and the other with h2 tag.